apps/server/lib/http/facades/action.ts

```diff
@@ -22,14 +27,14 @@ interface ActionFacadeOptions {
 	files?: FileDetails[];
 }
 
-type ActionPayload = Parameters<Producer['enqueue']>[2];
+type PreRequest = Parameters<Worker['pre']>[1];
```

Comment:
This should just be `ActionRequestData`? These two types are actually incompatible at the moment, but we should use a single type for both action request contracts and the function we use to pre-process action requests.